django/contrib/auth/decorators.py

```diff
@@ -50,6 +50,17 @@ def login_required(function=None, redirect_field_name=REDIRECT_FIELD_NAME, login
     return actual_decorator
 
 
+def login_not_required(view_func):
+    """
+    Decorator for views that marks that the view is accessible by
+    unauthenticated users.
+    """
+    def wrapped_view(*args, **kwargs):
+        return view_func(*args, **kwargs)
+    wrapped_view.login_not_required = True
```

Comment:
My vote for flipping `.login_not_required = True` around to `.login_required = False` to reduce cognitive load that seems unneeded. Has this been discussed?

Reply:
That was my bad. I don't even have an idea why I've choose `login_not_required` at all. I'll fix it this, thanks